Source/EasyNetQ/IServiceProvider.cs

```diff
@@ -49,5 +49,7 @@ public interface IServiceRegister
         IServiceRegister Register<TService, TImplementation>()
             where TService : class
             where TImplementation : class, TService;
+
+        IServiceRegister UseDelayedMessageExchange();
```

Comment:
Same here, I understand the purpose of these changes but I think the method to enable delayed exchanges shouldn't leave in `IServiceProvider`. It feels too unrelated.